src/mine_loop.rs

```diff
@@ -396,23 +398,20 @@ pub(crate) async fn make_coinbase_transaction_stateless(
     Ok((transaction, composer_outputs))
 }
 
-/// Compute `TransactionDetails` and a list of `TxOutput`s for a coinbase
-/// transaction.
-pub(super) fn prepare_coinbase_transaction_stateless(
-    latest_block: &Block,
+/// Produce two outputs spending a given portion of the coinbase.
+///
+/// The coinbase is the block subsidy plus the anticipated transaction fee.
+/// There are two outputs because one is liquid immediately, and the other is
+/// locked for 3 years. The portion is determined by the guesser fee fraction
```

Comment:
“The portion […].” The portion that is liquid immediately? Seems unlikely, but I'm not sure what other portion this might refer to.